youxia-q2seqware/src/main/java/io/cloudbindle/youxia/q2seqware/QueueListener.java

```diff
@@ -0,0 +1,9 @@
+package io.cloudbindle.youxia.q2seqware;
+
+public interface QueueListener extends SeqwareJobMessageSource {
```

Comment:
Remember to explain the purpose of each class (using Javadoc) and preferably all methods in interfaces.